cmd/ctr/ctr.go

```diff
@@ -37,6 +40,8 @@ func NewCtrCommand() *cobra.Command {
 		DisableFlagParsing: true,
 		RunE: func(_ *cobra.Command, _ []string) error {
 			args := extractCtrCommand(os.Args)
+			newPath := fmt.Sprintf("%s:%s/bin", config.GetCmdOpts().K0sVars.BinDir, os.Getenv(pathEnv))
```

Comment:
The colon could be replaced by [`os.PathListSeparator`](https://pkg.go.dev/os#PathListSeparator).

Reply:
Btw...  Is that trailing `/bin` right?